vendor/OWNERS

```diff
@@ -1,4 +0,0 @@
-# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
Please restore this file.

Reply:
+1, The dependencies CI job should have instructions in the failure output, this file is generated by a hack script and should continue to exist unmodified.

Reply:
Whoa, whoops. Looks like something around here dropped it on me by accident: https://github.com/kubernetes/kubernetes/blob/master/hack/update-vendor.sh#L362.

It's back!